components/brave_ads/core/internal/units/ad_type.cc

```diff
@@ -25,107 +26,67 @@ constexpr char kSearchResultAdType[] = "search_result_ad";
 
 }  // namespace
 
-AdType::AdType() = default;
```

Comment:
This is great that you took the time to do this, however we are currently in the process of transitioning these public facing data structures to mojom.

Reply:
The IDL mojom is at `components/brave_ads/core/mojom/brave_ads.mojom` so it would be great if we could add the enum here alongside the `ToString` change. Thanks